src/modules/validation_manager.cpp

```diff
@@ -0,0 +1,115 @@
+#include "die.h"
+#include "validation_manager.h"
+#include "config/board_config.h"
+#include "nrf_log.h"
+
+#include "animations/animation.h"
+#include "modules/anim_controller.h"
+#include "animations/animation_name.h"
+
+#include "bluetooth/bluetooth_messages.h"
+#include "bluetooth/bluetooth_message_service.h"
+#include "bluetooth/bluetooth_stack.h"
+
+#include "drivers_nrf/power_manager.h"
+#include "data_set/data_set.h"
+
+#include "utils/Utils.h"
+
+using namespace Animations;
+using namespace Modules;
+using namespace Config;
+using namespace DriversNRF;
+using namespace Bluetooth;
+
+#if !defined(FIRMWARE_VERSION)
+    #warning Firmware version not defined
+    #define FIRMWARE_VERSION "Unknown"
+#endif
+
+namespace Modules::ValidManager
+{
+	static AnimationName nameAnim;
+	bool isPlaying;
+
+	void skipFeed(void* token);
+	void stopNameAnim();
+	void startNameAnim();
```

Comment:
You have some tab here (only seeing this because in the alignment is off in GitHub review page)